auth-web/src/util/constants.ts

```diff
@@ -146,23 +145,22 @@ export enum LearFilingTypes {
     REGISTRATION = 'Registration'
 }
 
-export enum LegalTypes {
-    BCOMP = 'BEN',
+// NB: Corp Type is sometimes called Legal Type
+// see also https://github.com/bcgov/bcrs-shared-components/blob/main/src/modules/corp-type-module/corp-type-module.ts
+export enum CorpTypes {
+    // actual corp types
+    BC_CCC = 'CC',
+    BC_COMPANY = 'BC',
+    BC_ULC_COMPANY = 'ULC',
+    BENEFIT_COMPANY = 'BEN',
     COOP = 'CP',
-    CORP = 'CR',
-    SP = 'SP',
-    GP = 'GP'
-}
+    PARTNERSHIP = 'GP',
+    SOLE_PROP = 'SP',
 
-export enum CorpType {
```

Comment:
I renamed this enum (plural) to match other enums.

Also I renamed BCOMP -> BENEFIT_COMPANY (to match [CorpTypeCd](https://github.com/bcgov/bcrs-shared-components/blob/f3cfaed174852f89379435e888c435b2fe6e6516/src/modules/corp-type-module/corp-type-module.ts#L9).

It's unfortunate that this project uses both CorpTypeCd (imported) and CorpTypes, but CorpTypes includes some overloaded values for temp reg number affiliations, and it's out of scope to fix all that here.